driver/cassandra/README.md

```diff
@@ -5,6 +5,7 @@
 ```bash
 migrate -url cassandra://host:port/keyspace -path ./db/migrations create add_field_to_table
 migrate -url cassandra://host:port/keyspace -path ./db/migrations up
+CASSANDRA_CA_PATH=my_cassandra_ca.pem migrate -url cassandra://host:port/keyspace -path ./db/migrations up # Using encrypted connection
```

Comment:
let's not use env vars. `-url cassandra://host:port/keyspace?ca=my_cassandra_ca.pem` is the preferred way. not sure about the `ca` key though.